src/commands/game_commands/play.ts

```diff
@@ -148,7 +154,7 @@ export default class PlayCommand implements BaseCommand {
                 const startTitle = `\`${prefix}join\` the game and start it with \`${prefix}begin\`!`;
                 const gameInstructions = `Type \`${prefix}join\` to play in the upcoming elimination game. Once all have joined, ${getMention(gameOwner.id)} must send \`${prefix}begin\` to start the game. Everyone begins with \`${lives}\` lives.`;
 
-                gameSession = new GameSession(textChannel.id, voiceChannel.id, textChannel.guild.id, gameOwner, GameType.ELIMINATION, lives);
+                gameSession = new GameSession(textChannel.id, voiceChannel.id, textChannel.guild.id, gameOwner, GameType.ELIMINATION, isCompetitionMode, lives);
```

Comment:
I don't think it's possible to be both `competition` and `elimination | teams`?

Reply:
Any reason we don't make `GameType.COMPETITION` instead of new variable to keep track?

Reply:
> Any reason we don't make `GameType.COMPETITION` instead of new variable to keep track?

I can do this, but I will also need to refactor any CLASSIC checks in code and include competition there.

Once we enable argument validation here, if a user enters an invalid game type, the expected values shown will include competition even though it can't be used.

Reply:
> I don't think it's possible to be both `competition` and `elimination | teams`?

You're right. I put the variable there to show what the argument being passed was (instead of just writing false), but it can only be false with those game modes. If I refactor to make COMPETITION a GameType, I'll remove this